src/main/resources/client_secrets.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Could you remove the private key, id, client mail, id from git and set it on the server only?